gradio/buttons.py

```diff
@@ -0,0 +1,70 @@
+""" Predefined buttons with bound events that can be included in a gr.Blocks for convenience. """
+
+from __future__ import annotations
+
+import json
+
+from gradio_client.documentation import document, set_documentation_group
+from typing_extensions import Literal
+
+from gradio.components import Button, Component
+
+set_documentation_group("component")
+
+
+@document("add")
```

Comment:
why are we documenting `add`? Seems like an internal method.

Reply:
oh I guess if the cleared components come after huh

Reply:
Yea but I wonder how common that is outside of the `interface.py` code

Reply:
I don't have a strong opinion, but it might come in handy for complex demos so I've added a docstring and to the docs page.